libs/utils/energy.py

```diff
@@ -72,9 +73,13 @@
     'hikey' : {
         'instrument' : 'aep',
         'conf' : {
-            'labels'          : ['LITTLE'],
             'resistor_values' : [0.033],
             'device_entry'    : '/dev/ttyACM0',
+        },
+        # AEP requires channel_map for compatibility with the generic energy
+        # meters interface
+        "channel_map" : {
+            "LITTLE" : 'LITTLE',
```

Comment:
Yes, because below I pass `labels=aep_conf["channel_map"]`.... Oh so this means we cannot assign `channels` in the constructor